### DISCUSSION TITLE
Supabase Studio Won't Connect to "Default Project" (Self-Hosting)

### DISCUSSION THREAD

**Discussion by globemediaofficial** [supabase/supabase#9957] [Questions] [answered]
I have a pull from today and I configured everything as usual but now the studio won't connect to the project. All of the URL variables are set to `0.0.0.0:*port*`, just like they were before, and the studio opens with `0.0.0.0:3000`, it just says "Connecting to Default Project" endlessly.  Eventually it says "Still Connecting?" and then tells me to contact support. What could be the issue? I've double-checked all the URLs, passwords, and keys, they're al what they are meant to be.

**Comment by SamMakesThings** [comment, marked as answer]
Hey @globemediaofficial if you still need an answer to this, this helped me: https://github.com/supabase/supabase/discussions/7126

**Reply by SamMakesThings** [reply]
Specifically the part about replacing localhost with the machine's IP in .env
